src/main/java/net/glowstone/io/anvil/RegionFile.java

```diff
@@ -80,11 +77,13 @@ would be in region (0, -1), and one at (70, -30) would be at (3, -1).
     private static final int SECTOR_INTS = SECTOR_BYTES / 4;
 
     private static final int CHUNK_HEADER_SIZE = 5;
+    private static final int COMPRESSED_BUFFER_SIZE = 8192;
```

Comment:
This is just `SECTOR_BYTES * 2`. Is it really worth making a new constant? `SECTOR_BYTES * 2` is used throughout the code already.